packages/client/hmi-client/src/workflow/ops/model-config/tera-model-config.vue

```diff
@@ -18,41 +26,39 @@
 					/>
 				</div>
 				<div v-else-if="activeIndex === 1">
```

Comment:
No need for this div

Reply:
ive changed these in favour of v-show to keep the ui state when switching between pages.  So now the div is needed now!